.env.template

```diff
@@ -0,0 +1,3 @@
+WHITEFOX_TEST_AWS_REGION=eu-west-1
+WHITEFOX_TEST_AWS_SECRET_ACCESS_KEY=weashdfgvkeuajysmjtsrq2384we
+WHITEFOX_TEST_AWS_ACCESS_KEY_ID=AKIAYTRD345SDUY
```

Comment:
Why are you committing the whitefox credentials?

Reply:
these are fake credentials, it's a feature of dotenv plugin to help the user compile their own `.env` "real" file